_build/partials/interactive-map.hbs

```diff
@@ -0,0 +1,57 @@
+<div id="leaflet-map"></div>
+<script>
+    async function loadData(){
+        const pointsResponse = await fetch("{{geojsonPointsUrl}}");
+        const geojsonPoints = await pointsResponse.json();
+
+        const polygonsResponse = await fetch("{{geojsonPolygonsUrl}}");
+        const geojsonPolygons = await polygonsResponse.json();
+
+        return {geojsonPoints, geojsonPolygons };
+    }
+
+    function generatePopupText(feature){
+        const functionString = () => navigator.clipboard.writeText(JSON.stringify(feature.geometry))
+
+        return "<h4>title</h4>" +
+            "<div>" +
+            "Time range:" + feature.properties.from + " to " + feature.properties.to + "<br>" +
+            "Area size: " + feature.properties["area / sqkmm"] + " km<sup>2</sup><br>" +
+            '<button class="leaflet-copy-geojson">copy geojson</button>' +
```

Comment:
```suggestion
            '<button class="leaflet-copy-geojson">Copy GeoJSON</button>' +
```

Reply:
thanks, done

Reply:
Will add the icon when I will fix the button onclick function in a separate MR